src/main/java/com/thealgorithms/maths/Disarium.java

```diff
@@ -0,0 +1,39 @@
+package com.thealgorithms.maths;
+//if the sum of its power of the positions from left to right is equal to the number.
+//Example:
+//1^1 + 3^2 + 5^3 = 1 + 9 + 125 = 135
+ import java.util.*;
+
+public class Disarium {
+    public static void main(String args[])
```

Comment:
Do not put the algorithm inside the `main()` method create a different function `disarium` for the algorithm and call it from `main`